content/guides/farmlands-pos-integration.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Farmlands POS Integration Guide
+img: /farmlands-pos-integration-cover.jpg
```

Comment:
What was your reasoning for using a JPG over a PNG? I see the other image you used was a PNG

Reply:
@tom-blake I went to Unsplash and got the original image as the one from Figma wasn't large enough. Unsplash didn't have an option for the file type and downloaded it as a JPG.

Reply:
I see - thanks! Maybe let the designers know if Unsplash isn't what they'd do :)